src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/operators.ts

```diff
@@ -182,24 +216,36 @@ export async function getSuggestionsToRightOfOperatorExpression({
           finalType === 'boolean' &&
           getFunctionDefinition(operator.name)?.type === FunctionDefinitionTypes.OPERATOR
             ? ['any']
-            : (supportedTypes as string[]);
-
-        // TODO replace with fields callback + function suggestions
-        suggestions.push(
-          ...(await getFieldsOrFunctionsSuggestions(
-            typeToUse,
-            location,
-            getColumnsByType,
-            {
-              functions: true,
-              columns: true,
-              values: Boolean(operator.subtype === 'binary-expression'),
-            },
-            {},
-            hasMinimumLicenseRequired,
-            activeProduct
-          ))
-        );
+            : supportedTypes;
+
+        // Special case: if we have an incomplete unary operator that starts with "is", suggest the complete forms
+        const isUnaryIs = operator.name.startsWith('is ');
```

Comment:
So do I understand correctly that with your changes the is null, is not null broke and you have to handle it separately? Because in main is null works fine. 

If yes how does it break? I am asking because I dont get why it needs a different handling now

Reply:
if you look at the changes in the *.tests you also see the consistency of the results

Reply:
- Correct, They are operators.
- I removed the hardcoded 'is' and filtered the operators from the definitions (as we do in other cases).

Reply:
I saw it and approved already, thanx